src/AfriCC/EPP/Frame/Command/Update/Contact.php

```diff
@@ -7,6 +7,8 @@
  *
  * For the full copyright and license information, please view the LICENSE file
  * that was distributed with this source code.
+ *
+ * TODO: This class can generate content not compliant with RFC 5733. I wonder if we should force it to be compliant and rely on extensions for non-compliant registrars.
```

Comment:
👍 - did not know that, most prolly because I was using coza-registry as base.

Reply:
Generally the only add/rem thing in contact:change should be status. Everything else - chg.

I can create PR for full compliance but that would be Breaking Change.

Reply:
Oh interesting I see that now:

- even though my tests include add/rem for elements that break compliance, I do not think that there was an active use-case for that - meaning my tests are most prolly wrong (= I do not think there will be a breaking change per-se, unless you know of an extension that actively relies on this?)
- this is a interesting space because it means we are doing a bit more input validation (output validation is completely missing so far) - meaning: while I made things compliant, I don't think I have complete validations everywhere, so there might be other parts that can generate non-compliant xml as well :)

Reply:
Hmmmmm... You know... Let me make more PRs that I planned on and we'll see about breaking stuff. I think that making `contact:update` RFC compliant would also simplify the heck out of `AfriCC\EPP\Frame\Command\Update\Contact`